CONTRIBUTING.md

```diff
@@ -48,7 +48,6 @@ integration tests.
 
 ```bash
 export GOOGLE_APPLICATION_CREDENTIALS=/path/to/service/account.json
-export IT_SERVICE_ACCOUNT_EMAIL=my-service-account@my-project.gserviceaccount.com
```

Comment:
@dmitry-fa I know this was just updated. I want to get the README updated for now.

@chingor13 is there a way to customize CONTRIBUTING.md similar to README.md using synthtool?

Reply:
We don't right now. We can write that feature in synthtool/templates or ignore this file for regen.

Reply:
A feature could help, but ignoring CONTRIBUTING file seems reasonable for now. Do you have an example I could refer to?

Reply:
You can add to the excludes here: https://github.com/googleapis/java-storage/blob/master/synth.py#L22

Reply:
Thanks @chingor13